samples/snippets/src/main/java/com/example/spanner/PgSpannerSample.java

```diff
@@ -0,0 +1,702 @@
+        Arrays.asList(
+            Mutation.newUpdateBuilder("Albums")
+                .set("SingerId")
+                .to(1)
+                .set("AlbumId")
+                .to(1)
+                .set("MarketingBudget")
+                .to(100000)
+                .build(),
+            Mutation.newUpdateBuilder("Albums")
+                .set("SingerId")
+                .to(2)
+                .set("AlbumId")
+                .to(2)
+                .set("MarketingBudget")
+                .to(500000)
+                .build());
+    // This writes all the mutations to Cloud Spanner atomically.
+    dbClient.write(mutations);
+  }
+  // [END spanner_update_data]
+
+  // [START spanner_read_write_transaction]
+  static void writeWithTransaction(DatabaseClient dbClient) {
+    dbClient
+        .readWriteTransaction()
+        .run(transaction -> {
+          // Transfer marketing budget from one album to another. We do it in a transaction to
+          // ensure that the transfer is atomic.
+          Struct row =
+              transaction.readRow("Albums", Key.of(2, 2), Arrays.asList("MarketingBudget"));
+          long album2Budget = row.getLong(0);
+          // Transaction will only be committed if this condition still holds at the time of
+          // commit. Otherwise it will be aborted and the callable will be rerun by the
+          // client library.
+          long transfer = 200000;
+          if (album2Budget >= transfer) {
+            long album1Budget =
+                transaction
+                    .readRow("Albums", Key.of(1, 1), Arrays.asList("MarketingBudget"))
+                    .getLong(0);
+            album1Budget += transfer;
+            album2Budget -= transfer;
+            transaction.buffer(
+                Mutation.newUpdateBuilder("Albums")
+                    .set("SingerId")
+                    .to(1)
+                    .set("AlbumId")
+                    .to(1)
+                    .set("MarketingBudget")
+                    .to(album1Budget)
+                    .build());
+            transaction.buffer(
+                Mutation.newUpdateBuilder("Albums")
+                    .set("SingerId")
+                    .to(2)
+                    .set("AlbumId")
+                    .to(2)
+                    .set("MarketingBudget")
+                    .to(album2Budget)
+                    .build());
+          }
+          return null;
+        });
+  }
+  // [END spanner_read_write_transaction]
+
+  // [START spanner_query_data_with_new_column]
+  static void queryMarketingBudget(DatabaseClient dbClient) {
+    // Rows without an explicit value for MarketingBudget will have a MarketingBudget equal to
+    // null. A try-with-resource block is used to automatically release resources held by
+    // ResultSet.
+    try (ResultSet resultSet =
+             dbClient
+                 .singleUse()
+                 .executeQuery(Statement.of("SELECT singerid as \"SingerId\", "
```

Comment:
is the `singlerid as \"\SingerId"` used to keep the output looks same as before?

Or why not just  `SELECT SingerId, AlbumId, MarketingBudge FROM Albums`?